api/v1/services/TrackingService.js

```diff
@@ -31,7 +32,7 @@ module.exports.createTrackingEvent = function (attributes) {
             if(!_.isNil(result)) {
                 return cache.ttlAsync('trackedEvent')
                     .then(function (ttl) {
-                        var message = "An event is already being tracked at this time, seconds left: " + ttl;
+                        var message = "An event is already being tracked at this time, time left: " + ttl.toHMS();
```

Comment:
Thanks for the change, can we add the name of the event being tracked here too? Similar to line 63.